### ISSUE TITLE
TP8_EJ6

### ISSUE THREAD

**Issue by Jalbertonisalini** [marcelogarberoglio/PI#225]
Hola marcelo, como va? En este ejercicio de las racies no me anda ni la solucion propuesta por la catedra y no entiendo bien a que se debe. Me tira los siguientes errores:

esto en el caso de pasarle como el ejemplo un intervalo de [1-50000]:

malloc(): corrupted top size
Abortado (`core' generado)

esto en el caso de por ejemplo pasar [1-2]:

free(): invalid pointer
Abortado (`core' generado)

y esto en un caso de pasar [1-100]

corrupted size vs. prev_size
Abortado (`core' generado)

Es como un surtido de fallos y no entiendo bien a que se deben... :(

**Comment by marcelogarberoglio** [comment]
Ahí lo revisé. No me gusta mucho ese código, no lo reocrdaba, pero veo que están acopladas las funciones, sobre todo por el manejo de cant, y falla cuando hay más de 10 raíces (el valor de MEM_INC). Se puede emparchar pero mejor voy a hacer una nueva versión, donde esté mejor modularizado.

**Comment by Jalbertonisalini** [comment]
Dale perfecto! Vas a subir la solución nueva?

**Comment by marcelogarberoglio** [comment]
Sí, después subo una. Incluso está mal el comentario que dice "en materias posteriores...", ya que punteros a función también lo vemos en esta materia.
